Covid19Shahajjo/app/src/main/java/com/example/covid19shahajjo/activities/youtubeFullScreenActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.example.covid19shahajjo.activities;
+
+import android.os.Bundle;
+import android.view.View;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.covid19shahajjo.R;
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.PlayerConstants;
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.YouTubePlayer;
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.listeners.AbstractYouTubePlayerListener;
+import com.pierfrancescosoffritti.androidyoutubeplayer.core.player.views.YouTubePlayerView;
+
+public class youtubeFullScreenActivity extends AppCompatActivity {
```

Comment:
class name should start with Capital letter

Reply:
okk bhai .
updating